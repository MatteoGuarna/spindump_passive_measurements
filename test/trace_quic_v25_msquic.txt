A test with Msquic QUIC implementation server and Quant client.

There are four sessions:

- regular + full start
- regular + 0rtt 
- regular + h3
- regular + stateless retry
- key update
- quantum-mode

The commands used were:

   (rm /tmp/quant*;
   ./bin/client -i en0 https://quic.westus.cloudapp.azure.com:4433/5000 ;
   ./Bin/client -i en0 https://quic.westus.cloudapp.azure.com:4433/15000 ; 
   ./Bin/client -3 -i en0 https://quic.westus.cloudapp.azure.com:4433/15000 ; 
   ./Bin/client -i en0 https://quic.westus.cloudapp.azure.com:4434/15000 ;
   ./bin/client -u -i en0 https://quic.westus.cloudapp.azure.com:4433/10000 ;
   ./bin/client -m -i en0 https://quic.westus.cloudapp.azure.com:4433/20000 )


