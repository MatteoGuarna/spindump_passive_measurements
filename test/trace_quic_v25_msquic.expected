QUIC 100.72.46.172 <-> 138.91.188.147 6f960f91af-4adcbfcb (52169:4433) at 1580824407738282 new starting packets 0 0 bytes 0 0 note "No rsp,v.0xbabababa,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 6f960f91af-a44cd784124adc (52169:4433) at 1580824407913510 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v.0x00000000,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 6f960f91af-a44cd784124adc (52169:4433) at 1580824407913510 measurement starting right 175228 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v.0x00000000,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 null-a44cd784124adc (52169:4433) at 1580824407915835 measurement starting left 2325 packets 1 1 bytes 1228 69 bandwidth 1228 69 note "V25,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 null-08947c637c561f2e (52169:4433) at 1580824408091051 change starting packets 2 1 bytes 2456 69 bandwidth 2456 69 note "V25,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 null-08947c637c561f2e (52169:4433) at 1580824408091051 measurement up right 175216 packets 2 1 bytes 2456 69 bandwidth 2456 69 note "V25,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 3f853115-08947c637c561f2e (52169:4433) at 1580824408092115 packet up initiator length 1228 packets 3 2 bytes 3684 236 bandwidth 3684 236 note "V25,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 3f853115-08947c637c561f2e (52169:4433) at 1580824408251796 packet up responder length 1280 packets 3 3 bytes 3684 1516 bandwidth 3684 1516 note "V25,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 3f853115-08947c637c561f2e (52169:4433) at 1580824408253034 packet up responder length 338 packets 3 4 bytes 3684 1854 bandwidth 3684 1854 note "V25,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 3f853115-08947c637c561f2e (52169:4433) at 1580824408253480 packet up initiator length 72 packets 4 4 bytes 3756 1854 bandwidth 3756 1854 note "V25,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 3f853115-08947c637c561f2e (52169:4433) at 1580824408253772 packet up initiator length 71 packets 5 4 bytes 3827 1854 bandwidth 3827 1854 note "V25,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 3f853115-08947c637c561f2e (52169:4433) at 1580824408254916 packet up initiator length 1500 packets 6 4 bytes 5327 1854 bandwidth 5327 1854 note "V25,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 3f853115-08947c637c561f2e (52169:4433) at 1580824408575245 measurement up right 158770 packets 9 7 bytes 5504 4702 bandwidth 5504 4702 note "V25,spinning"
QUIC 100.72.46.172 <-> 138.91.188.147 04efb298-ea76151c025ae0f61d736fecc18bf369 (50349:4433) at 1580824408750751 new starting packets 0 0 bytes 0 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 04efb298-08ea85b678e64e08 (50349:4433) at 1580824409000075 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 04efb298-08ea85b678e64e08 (50349:4433) at 1580824409000075 measurement up right 249324 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 efb29899-08ea85b678e64e08 (50349:4433) at 1580824409002724 packet up initiator length 128 packets 2 1 bytes 1356 416 bandwidth 1356 416 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 efb29899-08ea85b678e64e08 (50349:4433) at 1580824409162357 packet up initiator length 1500 packets 3 3 bytes 2856 3196 bandwidth 2856 3196 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 efb29899-08ea85b678e64e08 (50349:4433) at 1580824409364073 measurement up full (responder) 17516 packets 5 4 bytes 3022 3255 bandwidth 3022 3255 note "V25,0-RTT,no I-spin"
QUIC 100.72.46.172 <-> 138.91.188.147 efb29899-08ea85b678e64e08 (50349:4433) at 1580824409407693 measurement up left 43620 packets 5 5 bytes 3022 3343 bandwidth 3022 3343 note "V25,0-RTT,spinning"
QUIC 100.72.46.172 <-> 138.91.188.147 efb29899-08ea85b678e64e08 (50349:4433) at 1580824409594368 measurement up right 186675 packets 8 7 bytes 3222 3470 bandwidth 3222 3470 note "V25,0-RTT,spinning"
QUIC 100.72.46.172 <-> 138.91.188.147 efb29899-08ea85b678e64e08 (50349:4433) at 1580824409594368 measurement up full (responder) 230295 packets 8 7 bytes 3222 3470 bandwidth 3222 3470 note "V25,0-RTT,spinning"
QUIC 100.72.46.172 <-> 138.91.188.147 0a1a49edcf795c8f517cfbee0d852e040d70-null (65407:4433) at 1580824409832242 new starting packets 0 0 bytes 0 0 note "No rsp,v.0xbabababa,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 0a1a49edcf795c8f517cfbee0d852e040d70-0d70d04f0f0a1a (65407:4433) at 1580824410024460 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v.0x00000000,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 0a1a49edcf795c8f517cfbee0d852e040d70-0d70d04f0f0a1a (65407:4433) at 1580824410024460 measurement starting right 192218 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v.0x00000000,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 null-0d70d04f0f0a1a (65407:4433) at 1580824410026754 measurement starting left 2294 packets 1 1 bytes 1228 66 bandwidth 1228 66 note "V25,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 null-0de4eed0615a71e55d1cd46cd5 (65407:4433) at 1580824410185986 change starting packets 2 1 bytes 2456 66 bandwidth 2456 66 note "V25,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 null-0de4eed0615a71e55d1cd46cd5 (65407:4433) at 1580824410185986 measurement up right 159232 packets 2 1 bytes 2456 66 bandwidth 2456 66 note "V25,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 04122bb5-a3d68f329412282c0c0c9231b1fab165b9 (61548:4434) at 1580824410359098 new starting packets 0 0 bytes 0 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 04122bb5-08a413de7d6b1531 (61548:4434) at 1580824410520607 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 122bb5df-08a413de7d6b1531 (61548:4434) at 1580824410522526 measurement starting left 1919 packets 1 1 bytes 1228 143 bandwidth 1228 143 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 122bb5df-08a42d9b8a6c3a78 (61548:4434) at 1580824410682740 change starting packets 2 1 bytes 2456 143 bandwidth 2456 143 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 122bb5df-08a42d9b8a6c3a78 (61548:4434) at 1580824410682740 measurement up right 160214 packets 2 1 bytes 2456 143 bandwidth 2456 143 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 122bb5df-08a42d9b8a6c3a78 (61548:4434) at 1580824410683721 packet up responder length 358 packets 2 3 bytes 2456 1781 bandwidth 2456 1781 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 122bb5df-08a42d9b8a6c3a78 (61548:4434) at 1580824410684826 packet up initiator length 153 packets 3 3 bytes 2609 1781 bandwidth 2609 1781 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 122bb5df-08a42d9b8a6c3a78 (61548:4434) at 1580824410685235 packet up initiator length 71 packets 4 3 bytes 2680 1781 bandwidth 2680 1781 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 122bb5df-08a42d9b8a6c3a78 (61548:4434) at 1580824410686011 packet up initiator length 208 packets 5 3 bytes 2888 1781 bandwidth 2888 1781 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 122bb5df-08a42d9b8a6c3a78 (61548:4434) at 1580824410845171 packet up initiator length 1500 packets 6 5 bytes 4388 4561 bandwidth 4388 4561 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 122bb5df-08a42d9b8a6c3a78 (61548:4434) at 1580824411049018 measurement up full (responder) 19554 packets 8 6 bytes 4554 4620 bandwidth 4554 4620 note "V25,0-RTT,no I-spin"
QUIC 100.72.46.172 <-> 138.91.188.147 122bb5df-08a42d9b8a6c3a78 (61548:4434) at 1580824411090658 measurement up left 41640 packets 8 7 bytes 4554 4708 bandwidth 4554 4708 note "V25,0-RTT,spinning"
QUIC 100.72.46.172 <-> 138.91.188.147 122bb5df-08a42d9b8a6c3a78 (61548:4434) at 1580824411273420 measurement up right 182762 packets 11 9 bytes 4754 4835 bandwidth 4754 4835 note "V25,0-RTT,spinning"
QUIC 100.72.46.172 <-> 138.91.188.147 122bb5df-08a42d9b8a6c3a78 (61548:4434) at 1580824411273420 measurement up full (responder) 224402 packets 11 9 bytes 4754 4835 bandwidth 4754 4835 note "V25,0-RTT,spinning"
QUIC 100.72.46.172 <-> 138.91.188.147 122bb5df-08a42d9b8a6c3a78 (61548:4434) at 1580824411350339 measurement up left 76919 packets 11 11 bytes 4754 4955 bandwidth 4754 4955 note "V25,0-RTT,spinning"
QUIC 100.72.46.172 <-> 138.91.188.147 122bb5df-08a42d9b8a6c3a78 (61548:4434) at 1580824411350339 measurement up full (initiator) 259681 packets 11 11 bytes 4754 4955 bandwidth 4754 4955 note "V25,0-RTT,spinning"
QUIC 100.72.46.172 <-> 138.91.188.147 04205f7b-95aa2988cbcba18695c5f8d5c5 (49750:4433) at 1580824411528029 new starting packets 0 0 bytes 0 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 04205f7b-089524336b3ecfbf (49750:4433) at 1580824411764975 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 04205f7b-089524336b3ecfbf (49750:4433) at 1580824411764975 measurement up right 236946 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 205f7be4-089524336b3ecfbf (49750:4433) at 1580824411765347 packet up responder length 337 packets 1 2 bytes 1228 1617 bandwidth 1228 1617 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 205f7be4-089524336b3ecfbf (49750:4433) at 1580824411767276 packet up initiator length 72 packets 2 2 bytes 1300 1617 bandwidth 1300 1617 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 205f7be4-089524336b3ecfbf (49750:4433) at 1580824411767670 packet up initiator length 71 packets 3 2 bytes 1371 1617 bandwidth 1371 1617 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 205f7be4-089524336b3ecfbf (49750:4433) at 1580824411768458 packet up initiator length 128 packets 4 2 bytes 1499 1617 bandwidth 1499 1617 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 205f7be4-089524336b3ecfbf (49750:4433) at 1580824411922313 packet up initiator length 1500 packets 5 4 bytes 2999 4397 bandwidth 2999 4397 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 205f7be4-089524336b3ecfbf (49750:4433) at 1580824412118911 measurement up full (responder) 15731 packets 7 5 bytes 3165 4456 bandwidth 3165 4456 note "V25,0-RTT,no I-spin"
QUIC 100.72.46.172 <-> 138.91.188.147 205f7be4-089524336b3ecfbf (49750:4433) at 1580824412160830 measurement up left 41919 packets 7 6 bytes 3165 4544 bandwidth 3165 4544 note "V25,0-RTT,spinning"
QUIC 100.72.46.172 <-> 138.91.188.147 205f7be4-089524336b3ecfbf (49750:4433) at 1580824412341275 measurement up right 180445 packets 10 8 bytes 3365 4671 bandwidth 3365 4671 note "V25,0-RTT,spinning"
QUIC 100.72.46.172 <-> 138.91.188.147 205f7be4-089524336b3ecfbf (49750:4433) at 1580824412341275 measurement up full (responder) 222364 packets 10 8 bytes 3365 4671 bandwidth 3365 4671 note "V25,0-RTT,spinning"
QUIC 100.72.46.172 <-> 138.91.188.147 205f7be4-089524336b3ecfbf (49750:4433) at 1580824412418237 measurement up left 76962 packets 10 10 bytes 3365 4791 bandwidth 3365 4791 note "V25,0-RTT,spinning"
QUIC 100.72.46.172 <-> 138.91.188.147 205f7be4-089524336b3ecfbf (49750:4433) at 1580824412418237 measurement up full (initiator) 257407 packets 10 10 bytes 3365 4791 bandwidth 3365 4791 note "V25,0-RTT,spinning"
QUIC 100.72.46.172 <-> 138.91.188.147 04e898cf-e540eb84e8fcbdaa4b1619d2 (63321:4433) at 1580824412579427 new starting packets 0 0 bytes 0 0 note "No rsp,v25,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 04e898cf-e540eb84e8fcbdaa4b1619d2 (63321:4433) at 1580824412579429 packet starting initiator length 613 packets 2 0 bytes 1893 0 bandwidth 1893 0 note "No rsp,v25,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 04e898cf-0ce540eb84e8fcbdaa4b1619 (63321:4433) at 1580824412789417 change starting packets 2 0 bytes 1893 0 bandwidth 1893 0 note "No rsp,v25,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 04e898cf-0ce540eb84e8fcbdaa4b1619 (63321:4433) at 1580824412789417 measurement up right 209988 packets 2 0 bytes 1893 0 bandwidth 1893 0 note "No rsp,v25,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 e898cf23-0ce540eb84e8fcbdaa4b1619 (63321:4433) at 1580824413831983 packet up initiator length 1228 packets 3 1 bytes 3121 80 bandwidth 1893 80 note "V25,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 e898cf23-0ce540eb84e8fcbdaa4b1619 (63321:4433) at 1580824416337274 packet up initiator length 1228 packets 4 1 bytes 4349 80 bandwidth 1228 80 note "V25,no spin"
QUIC 100.72.46.172 <-> 138.91.188.147 e898cf23-0ce540eb84e8fcbdaa4b1619 (63321:4433) at 1580824421344400 packet up initiator length 1228 packets 5 1 bytes 5577 80 bandwidth 1228 80 note "V25,no spin"
